### ISSUE TITLE
Genetic Map in diploid species

### ISSUE THREAD

**Issue by bechee** [mmollina/MAPpoly#72]
Hello. juts wondering if MAPpoly works well to construct genetic map for diploid markers?

Let's say I have the markers coded as 0,1, 2 = AA, AB, BB in this case,  0 would be AA, 1 would be AB, and 2 would be BB. Am I right?

Thanks,

Eduardo

**Comment by jeekinlau** [comment]
Yes it works well with diploid outcrossing species (cannot be inbred lines mapping). Make sure when you import the data it is specified that you are working with a diploid. ploidy=2

**Comment by bechee** [comment]
OK, why does not working for inbred lines? I have an F2 population

**Comment by jeekinlau** [comment]
Mappoly was designed for autopolyploids and I can't think of any inbred autopolyploid crop. If you have inbred diploid populations use OneMap which have inbred models built in.

Polysomic inheritance makes it virtually impossible to self all the way down to an inbred line. The only feasible way to get an autopolyploid inbred is to chromosomal doubling of a doubled haploid (outcrossing diploid species) or by chromosomal doubling of a diploid inbred line.

Hope that answers your question. Let us know if you have more

Jeekin
